### PR TITLE
🍒 feat(grid): show border for column resize

### PR DESCRIPTION
**Pull request by dan-mirescu**
Show colored border for column resize instead of a drag icon.

https://uipath.atlassian.net/browse/PLT-29993

It is a prerequisite for the horizontally scrollable grid feature.

### CODE REVIEW

File: projects/angular/components/ui-grid/src/ui-grid.component.scss [hunk 1]
```diff
@@ -365,10 +368,11 @@ ui-grid {
             display: block;
 
             .ui-grid-header-title {
-                width: calc(100% - #{$ui-header-resize-handle-width + $ui-grid-default-spacing});
+                width: 100%;
                 height: 100%;
                 display: inline-flex;
                 align-items: center;
+                position: relative;
```
Comment (gheorghitahurmuz) [inline projects/angular/components/ui-grid/src/ui-grid.component.scss:375]: why do we need position relative?

File: projects/angular/components/ui-grid/src/ui-grid.component.scss [hunk 2]
```diff
@@ -438,6 +442,23 @@ ui-grid {
                 }
             }
 
+            .ui-grid-cell-resizing-border {
+                flex: 0 0 $ui-cell-resizing-border-width;
+                height: $ui-grid-row-height + $ui-row-border-width;
+                margin-top: -$ui-row-border-width;
```
Comment (gheorghitahurmuz) [inline projects/angular/components/ui-grid/src/ui-grid.component.scss:448]: nit: negative values like `-1 * $ui-row-border-width` may be more readable

**Review by gheorghitahurmuz — APPROVED** [review]

File: projects/angular/components/ui-grid/src/ui-grid.component.scss [hunk 3]
```diff
@@ -438,6 +442,23 @@ ui-grid {
...
+            }
+
+            .ui-grid-header-cell {
+                &.ui-grid-spacer-cell {
+                    min-width: $ui-feature-cell-width;
+                }
+            }
+
+            .ui-grid-header-cell {
+                justify-content: space-between;
```
Comment (caleniuc) [inline projects/angular/components/ui-grid/src/ui-grid.component.scss:459]: In what case do we need this?

nit: can be placed inside the above rule, just for `.ui-grid-header-cell`

**Review by caleniuc — APPROVED** [review]

File: projects/angular/components/ui-grid/src/ui-grid.component.scss (hunk 1 above)
Comment (dan-mirescu) [reply projects/angular/components/ui-grid/src/ui-grid.component.scss:375]: It seems it's not needed. I will remove it.

**Pull request merged**